Code/Framework/AzFramework/AzFramework/DocumentPropertyEditor/Reflection/LegacyReflectionBridge.cpp

```diff
@@ -824,19 +847,40 @@ namespace AZ::Reflection
                 // Prepare the node references for the handlers.
                 StackEntry& parentData = m_stack.back();
 
+                const AZ::Edit::ElementData* elementEditData = (classElement ? classElement->m_editData : nullptr);
                 if (classElement)
```

Comment:
Looks like this can be simplified:

```suggestion
const AZ::Edit::ElementData* elementEditData = nullptr;
if (classElement)
{
    elementEditData = classElement->m_editData;
    ...
}
```